__tests__/gui/tabular_comparison/tabular_comparison.js

```diff
@@ -66,7 +66,12 @@ Then('I check result table after adding second indicator', (type) => {
    cy.get(`[data-testid=result-table]`).find('th').eq(2).should('have.text', 'Employment status : Unemployed');
    cy.get(`[data-testid=result-table]`).find('tbody tr').eq(0).find('td').eq(0).should('have.text', 'Eastern Cape');
    cy.get(`[data-testid=result-table]`).find('tbody tr').eq(0).find('td').eq(1).should('have.text', '301,909');
-   cy.get(`[data-testid=result-table]`).find('tbody tr').eq(0).find('td').eq(2).should('have.text', '952,709');
+   cy.get(`[data-testid=result-table]`).find('tbody tr').eq(0).find('td').eq(1).should('have.css', 'background-color').and('eq', 'rgb(124, 162, 206)')
+   cy.get(`[data-testid=result-table]`).find('tbody tr').eq(0).find('td').eq(2).should('have.text', '17.02%');
+   cy.get(`[data-testid=result-table]`).find('tbody tr').eq(0).find('td').eq(2).should('have.css', 'background-color').and('eq', 'rgb(124, 162, 206)')
+   cy.get(`[data-testid=result-table]`).find('tbody tr').eq(0).find('td').eq(2).trigger('mouseover');
+   cy.get(`[role="tooltip"]`).first().find('div').should('have.text', '952,709');
+   cy.get(`[data-testid=result-table]`).find('tbody tr').eq(0).find('td').eq(2).trigger('mouseout');
```

Comment:
Is it useful to trigger the mouseout even here? Should we then check that the tooltip is hidden?

Reply:
Initially i just did mouse over for both of cols but second tooltip didn't work correctly. so i used mouse out event to simulate proper behaviour.
We are using tooltip of MUI so i don't think we need to test showing and hiding of tooltip.

Reply:
@michaelglenister